_includes/seo.html

```diff
@@ -0,0 +1,31 @@
+    {% if page.title %}
+      {% capture seo_title %}{{ page.title | markdownify | strip_html | strip_newlines }} - {{ site.title }}{% endcapture %}
+    {% else %}
+      {% assign seo_title = site.title %}
+    {% endif %}
+
+    {% if page.description %}
+      {% assign seo_description = page.description | strip_html | strip_newlines %}
+    {% else %}
+      {% assign seo_description = site.description | strip_html | strip_newlines %}
+    {% endif %}
+
+    <!-- le seo -->
+    <title>{{ seo_title }}</title>
+    <meta property="og:title" content="{{ seo_title }}"/>
+    <meta property="og:site_name" content="{{ site.title }}"/>
+
+    <meta name="description" content="{{ seo_description }}">
+    <meta property='og:description' content="{{ seo_description }}"/>
+
+    <link rel="canonical" href="{{ site.github.url }}{{ page.url }}"/>
+    <meta property='og:url' content='{{ site.github.url }}{{ page.url }}'/>
+    <meta property='og:locale' content='en_us'/>
+
+    <meta property="twitter:account_id" content="{{ site.twitter.account_id }}" />
```

Comment:
The Twitter docs use `name` instead of `property`. It doesn't make any difference to them as far as I can tell, but 3rd party consumers/validators sometimes complain about it i.e. consider using

```html
<meta name="twitter:card" content="summary" />
```

instead of 

```html
<meta property="twitter:card" content="summary" />
```

if you want to be a perfectionist :ok_hand: